components/db/operations.ts

```diff
@@ -26,3 +28,10 @@ export async function listBills(
   const result = await getDocs(q)
   return result.docs.map(d => d.data()) as any
 }
+
+export async function getBill(id: string): Promise<Bill> {
+  const bill = await getDoc(
+    doc(firestore, `/generalCourts/${currentGeneralCourt}/documents/${id}`)
```

Comment:
Is currentGeneralCourt a parameter that is passed or is that a global variable?

Reply:
currently it's a global -- see line 14. The scraper also only handles the current court.